packages/cli/src/locale/strip-locale.ts

```diff
@@ -0,0 +1,104 @@
+import { unflatten } from 'flat';
+import fs from 'fs';
+import get from 'lodash/get';
+import path from 'path';
+import { Node, Project, PropertyAssignment, SyntaxKind } from 'ts-morph';
+
+// These keys aren't directly referenced in the code base, so we add them manually here
+const whitelist = [
+  'choropleth.tested_overall',
+  'choropleth.escalation_levels',
+  'choropleth.hospital_nice',
+  'choropleth.nursing_home',
+  'choropleth.disability_care',
+  'choropleth.elderly_at_home',
+  'choropleth.sewer',
+  'choropleth.behavior',
+];
+
+const APP_LOCALE_DIR = path.join(__dirname, '../../../app/src/locale');
+
+const project = new Project({
+  tsConfigFilePath: path.join(__dirname, '../../../app/tsconfig.json'),
+});
+
+const NlOriginal = JSON.parse(
+  fs.readFileSync(path.join(APP_LOCALE_DIR, 'nl.json'), {
+    encoding: 'utf-8',
+  })
+);
+const EnOriginal = JSON.parse(
+  fs.readFileSync(path.join(APP_LOCALE_DIR, 'en.json'), {
+    encoding: 'utf-8',
+  })
+);
+
+const sourceFile = project.getSourceFile('nl.json');
+
+const propertyAssignmentNodes: PropertyAssignment[] = (
+  sourceFile?.getDescendantsOfKind(SyntaxKind.PropertyAssignment) ?? []
+)
+  // each assignment always has at least one reference (the one in the file where its written),
+  // so here we only want the ones that have more than one ref. (The ones who are referenced in some other file(s))
+  .filter((x) => x.findReferences().length > 1);
+
+const newLocaleObjects = propertyAssignmentNodes
+  .map((x) => createFullPropertyChain(x))
+  .concat(whitelist)
+  // Only keep the deepest paths:
+  .filter(
+    (x, _i, l) =>
+      l.findIndex((y) => y.startsWith(`${x}.`) && y.length > x.length + 1) ===
+      -1
+  )
```

Comment:
nit; I think one letter variable names should be avoided. With the exception of x because we standardized that for short-hand purposes. But in this case x is not so obvious so I'd suggest to name that too, with the benefit of being able to use x to replace y. I'm using `subject` here not sure if that's the best name for it, or I understand the code correctly, but I think it's the same as what subject is in sanity LokalizeText documents...

```suggestion
    (subject, _index, list) =>
      list.findIndex((x) => x.startsWith(`${subject}.`) && x.length > x.length + 1) ===
      -1
  )
```